Specify the option to select when installing

andynava opened the pull request:
The UI has it as "Azure development" 
https://docs.microsoft.com/en-us/azure/azure-functions/media/functions-create-your-first-function-visual-studio/functions-vs-workloads.png

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

andynava committed:
Specify the option to select when installing

The UI has it as "Azure development" 
https://docs.microsoft.com/en-us/azure/azure-functions/media/functions-create-your-first-function-visual-studio/functions-vs-workloads.png

cgillum reviewed (approved):
#sign-off
